src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <star class="stars__star"></star>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```
just use regular divs, there is no such tag as star